README.md

```diff
@@ -76,6 +76,7 @@ data into vector tiles that can be rendered dynamically.
 - [OGR MVT](http://gdal.org/drv_mvt.html) and [MBTiles](http://gdal.org/frmt_mbtiles.html) - The GDAL/OGR MVT and MBTILES drivers can be used to read and write vector tiles, respectively as tileset on the filesystem or in a mbtiles container (GDAL >= 2.3.0)
 - [MBUtil](https://github.com/mapbox/mbutil) - Import and export MBTiles to disk :warning: no longer maintained
 - [Datamaps](https://github.com/ericfischer/datamaps) C application that can be used to create vector tiles and store them in an mbtiles. See the `render-vector` command. :warning: no longer maintained, use tippecanoe instead
+- [vt2geojson @wangyoucao577](https://github.com/wangyoucao577/vt2geojson) Command line tool to convert Vector Tiles to GeoJSON (written in `Go` language based on the awesome [orb](https://github.com/paulmach/orb) package).    
```

Comment:
For consistency, can you remove `@wangyoucao577` from the link title?

Reply:
I add the name because it's duplicate with another `vt2geojson` as above. Do I still need remove the name? If so, I'll do the change. Thanks!

Reply:
Oh, sorry. I'll do the renaming. It seems no confliction.

Reply:
Done.